cheat.lua

```diff
@@ -1,3 +1,22 @@
+function HandlePowertoolCommand(Split, Player)
+       	if not(Split[2] == nil) then
```

Comment:
Rather than using this use this:
```lua
if (not Split[2]) then
   Player:SendMessageFailure("Specify a command")
   return true
end
-- Other code
```